pkgs/tools/video/tsduck/default.nix

```diff
@@ -0,0 +1,54 @@
+{ lib, stdenv, fetchFromGitHub
+, curl
+, jdk
+, libedit
+, srt
+}:
+
+stdenv.mkDerivation rec {
+  pname = "tsduck";
+  version = "v3.31-2761";
+
+  src = fetchFromGitHub {
+    owner = "tsduck";
+    repo = pname;
+    rev = version;
+    sha256 = "sha256-268TKCh3naebbw+sOQ6d4N/zl7UEVtc3l3flFAYHDU4=";
+  };
+
+  buildInputs = [
+    curl
+    libedit
+    srt
+    jdk
+  ];
+
+  enableParallelBuilding = true;
+  makeFlags = [
+    "NODEKTEC=1"
+    "NOHIDES=1"
+    "NOPCSC=1"
+    "NORIST=1"
+    "NOVATEK=1"
+  ] ++ installFlags;
+
+  doCheck = false;  # calls out to https://tsduck.io/download/test/...
```

Comment:
That seems to be the case for 4 tests out of a large suite. Could you disable those tests instead?

Reply:
For example before testing, substitute:
```
    TSUNIT_TEST(testMasterPlaylist);
    TSUNIT_TEST(testMasterPlaylistWithAlternate);
    TSUNIT_TEST(testMediaPlaylist);
```
with empty text in https://github.com/tsduck/tsduck/blob/eddb9c7a4f7f3aeac40713758f550c98426b148d/src/utest/utestHLS.cpp#L57-L59

Reply:
Very good point, thank you.

I've added directives to remove the failing tests in the patch phase.

Fixed up and respun with all changes.